app/Rules/Yaml.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
I think this file could probably be renamed to `IsYaml` which is more correct for what it is validating, and would certainly reduce the confusion with the Symfony YAML package.